### PR TITLE
Uninstall module used on PS 1.6 before using this one

### PR DESCRIPTION
**Pull request by Quetzacoalt91**

### CODE REVIEW

File: ps_imageslider.php [hunk 1]
```diff
@@ -70,57 +75,130 @@ public function __construct()
...
 
-            /* Creates tables */
-            $res &= $this->createTables();
+        // Disable on mobiles and tablets
+        $this->disableDevice(Context::DEVICE_MOBILE);
 
-            /* Adds samples */
-            if ($res) {
-                $this->installSamples();
+        return (bool)$res;
```
Comment (PierreRambaud) [inline ps_imageslider.php]: $res is already a boolean because of `$res &=` :thinking:

Comment (Quetzacoalt91) [reply ps_imageslider.php]: Updated. That was coming from the previous version, that's why I did not change it.

**Comment by sarahdib** [comment]
after upgrading from 1.6.1.14 to 1.7.6.5 the module isn't upgrade

**Comment by matks** [comment]
@Quetzacoalt91 As I explained [in this issue](https://github.com/PrestaShop/PrestaShop/issues/26973) I will clean our open PRs backlog and close these stale PRs. But I hope that one day we get some bandwidth to reopen this topic, and when it happens, we'll reopen these PRs, find the issues noticed by QA, fix them and merge them ❤️  .

**Pull request closed without merging**